.github/workflows/linting.yml

```diff
@@ -0,0 +1,33 @@
+name: Linting
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    paths:
+      - '**/**.ts'
+      - '**/**.tsx'
+      - '**/package.json'
+      - '**/bun.lockb'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        project: ['upstream-tag-on-merge', 'upstream-tag-sync']
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v1
```

Comment:
Is there a reason we use `v1` here but `v2` in `.github/workflows/building.yml`?

Reply:
Good catch!